Web/Views/Candidates/Details.cshtml

```diff
@@ -67,18 +67,18 @@
                 @if (detail.Format == CandidateDetailFormat.List)
                 {
                     <ul>
-                        <li>@Html.DisplayFor(model => detail.Text)</li>
+                        <li>@Html.Raw(detail.Text)</li>
```

Comment:
Raw can be dangerous. Be careful!

Reply:
confirmed it with medhat during tuesday lab.